.github/workflows/coverage.yml

```diff
@@ -0,0 +1,47 @@
+name: Coverage
+
+on:
+  pull_request:
+    branches:
+    - "main"
```

Comment:
indent

Reply:
ok to fix these inconsistencies in a better way, have a PR for you https://github.com/checkdigit/prettier-config/pull/11